src/tunacode/ui/commands/model.py

```
@@ -0,0 +1,126 @@
+
+from tunacode.ui.commands.base import Command
+
+if TYPE_CHECKING:
+    from tunacode.ui.app import TextualReplApp
+
+
+def _validate_provider_api_key_with_notification(
+    model_string: str,
+    user_config: dict,
+    app: TextualReplApp,
+    show_config_path: bool = False,
```

Ruff (0.15.0)

[error] Boolean-typed positional argument in function definition

(FBT001)